templates/common/_base/files/additional-trust-bundle.yaml

```diff
@@ -0,0 +1,8 @@
+filesystem: "root"
+mode: 0600
+path: "/etc/pki/ca-trust/source/anchors/ca.crt"
```

Comment:
Super nit but...it'd be a lot more obvious if this was named `openshift-config-user-ca-bundle.crt` or so.

Reply:
ack, I thought naming was `ca.crt` by default, changing!

Reply:
fixed!